index.js

```diff
@@ -1,29 +1,11 @@
+const EventEmitter = require('events');
```

Comment:
IMHO, update test matrix and drop Node 0.12 (see https://github.com/ember-cli/capture-exit/pull/23#issuecomment-318862262).